### PR TITLE
Allow configuration of channels per producer/consumer connection

### PR DESCRIPTION
**Pull request by simonjjones**
To allow simulating load arriving across channels on connections

### CODE REVIEW

File: src/main/java/com/rabbitmq/perf/MulticastParams.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
     private long confirm = -1;
     private int consumerCount = 1;
     private int producerCount = 1;
+    private int consumerChannels = 1;
```
Comment (michaelklishin) [inline src/main/java/com/rabbitmq/perf/MulticastParams.java]: We use `*Count` names above for similar numerical values.

**Comment by michaelklishin** [comment]
@simonjjones can you please explain a use case for this? Sure, many connections use multiple channels but how often does it make a difference for throughput (unless you have a single connection)?

**Comment by acogoluegnes** [comment]
@simonjjones Thank you!

**Comment by gerhard** [comment]
@michaelklishin it's exactly that, to demonstrate whether multiple channels per connection make a difference.

File: src/main/java/com/rabbitmq/perf/MulticastParams.java (hunk 1 above)
Comment (gerhard) [inline src/main/java/com/rabbitmq/perf/MulticastParams.java]: Agreed with Simon to call this `consumerChannelsPerConnection`. Would you prefer we used `consumerChannelsPerConnectionCount`?

Comment (acogoluegnes) [inline src/main/java/com/rabbitmq/perf/MulticastParams.java]: We ended up with `producer/consumerChannelCount` in https://github.com/rabbitmq/rabbitmq-perf-test/commit/624e6caddf449a3d56b509b62fa545d8e5155c37. This is enough for consistency sake, the options descriptions being descriptive.

**Comment by gerhard** [comment]
@simonjjones can you re-build the [JAR](https://github.com/pivotal-cf-experimental/rabbitmq-perf-test-cf-wrapper/blob/master/pom.xml#L17-L21) from this repo and confirm that the changes are working for you?

**Pull request merged**